frontend/App.js

```diff
@@ -1,14 +1,10 @@
 // @flow
-import createStore from 'unistore';
-import { Provider } from 'unistore/react';
-
 type Props = {
-    data: {},
+    data: {
+        CAPI: CAPIType,
```

Comment:
Is there a way to configure things such that you have to explicitly import the type? And if so, is that a good idea?

I wonder whether it would make things clearer. It feels like a secret import at the moment.

Reply:
Deferring a decision on this until our meeting this week on types